.github/workflows/ci.yml

```
@@ -0,0 +1,19 @@
+name: CI
+on: [push, pull_request]
+jobs:
+
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Install Go
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.13
+
+    - name: Check out code
+      uses: actions/checkout@v1
+
+    - name: Run tests
+      run: go test ./...
```

could be useful to have this more verbose:
```suggestion
      run: go test ./... -v
```